### DISCUSSION TITLE
Notifications for both Mobile and Web Portal

### DISCUSSION THREAD

**Discussion by pujanthakkar** [The-Commit-Company/raven#605] [General]
We have started using Raven and we are loving it, but without notifications / alerts it isvery difficult for the team to adapt.

**Comment by nikkothari22** [comment]
Yeah, same here. Hence, notifications is a priority feature and we're currently building it. 

It's challenging though since we need to make tooling that does not exist within Frappe framework itself.
The feature is being tracked on #547

**Comment by BaaridunNasr** [comment]


**Reply by nikkothari22** [reply]
You need to enable Push Notifications on your Frappe site (search for Push Notification Settings in the Desk interface)

Refresh Raven, and then in the bottom left corner, the three dot menu will have an option for "Enable Push Notifications". Enable that and give the required permissions to the site. 

For each channel, in channel settings, you can enable push notifications.

Note that this will only run on Frappe Cloud for now.

**Reply by nikkothari22** [reply]
Can you provide your site name? I'll have a look in the relay server. 

You might want to toggle off and on the push notification settings in the existing channels. 

Also, check for DMs

**Reply by nikkothari22** [reply]
@PhantomKunai No limits as such AFAIK.

**Reply by dawoodjee** [reply]
Wondering how to activate this on a selfhosted site or via mobile. 

How do I setup the mobile app? 

edit: answer is at the top of this thread. not sure how i missed it

**Reply by sagarpatel08** [reply]
> You need to enable Push Notifications on your Frappe site (search for Push Notification Settings in the Desk interface)
> 
> Refresh Raven, and then in the bottom left corner, the three dot menu will have an option for "Enable Push Notifications". Enable that and give the required permissions to the site.
> 
> For each channel, in channel settings, you can enable push notifications.
> 
> Note that this will only run on Frappe Cloud for now.

I have tried all of that, but i still am not getting push notifications on Raven Desktop or ERP NEXT
